metadata/metadata.go

```diff
@@ -14,19 +14,28 @@
 
 package metadata
 
-import "errors"
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/mendersoftware/log"
+)
+
+type Validater interface {
```

Comment:
Go convention is quite strict about this. If you have an interface with only one method its name should be `method name + (e)r` so `Validater` in this case.